UNRELEASED.md

```diff
@@ -0,0 +1,36 @@
+# Unreleased
+
+> Place your changes below this line.
+>
+
+**Fixed:**
+- `BpkSwitch`:
```

Comment:
This should be `bpk-component-calendar`?

Reply:
As this also affects the scrollable calendar and the datepicker - might be good to include them in here too

Reply:
Oops, I forgot to change it. Addressed by d22ae113972fcdb3d787b76a256c8d4b26258ca4.

Reply:
Also by d0149658957af35617bfe76ac3c2651111733b97.